ext/Pod-Html/lib/Pod/Html.pm

```diff
@@ -842,4 +834,53 @@ sub relativize_url {
     return $rel_path;
 }
 
+sub get_header {
+	my $mode = shift() // "html";
+	my ($title, $csslink, $bodyid, $block) = @_;
```

Comment:
This is very odd to me.

1. You're sending the mode (`html`/`xhtml`) in the only call to this function, so it's pointless to check `// "html"`.
2. You're taking out the title, css link, body ID, and block immediately afterward, which is used by both modes.

I can only imagine one situation in which you would lack a mode but have correct values for the other variables - when you are sending `undef` to the function as the first argument. I can't see this happening.

Reply:
You're 100% right on this... this is an easy fix.